app/services/google_translation_service.rb

```diff
@@ -73,9 +88,42 @@ def transform_translations_for_caching(listing_id, keys, translations, last_modi
     return_value
   end
 
-  def google_translation_logger(message, error: false)
+  def log_text_to_translate(caller_method, targets, text, field_names, listing_id)
```

Comment:
Couldn't these two be one method, with a param for the log (that's the only difference?)

Reply:
These actually also don't have to be pulled out into methods, bc they are only used once each